stl/inc/random

```diff
@@ -694,8 +694,9 @@ public:
         seed(_Gx);
     }
 
-    void seed(_Seed_t _Value = default_seed, bool _Readcy = false) { // set initial values from specified seed value
-        linear_congruential<_Seed_t, 40014U, 0U, 2147483563U> _Lc{_Value == 0U ? default_seed : _Value};
+    void seed(_Seed_t _Value = 0u, bool _Readcy = false) { // set initial values from specified seed value
+        linear_congruential_engine<uint_least32_t, 40014U, 0U, 2147483563U> _Lc{
+            static_cast<uint_least32_t>(_Value == 0U ? default_seed : _Value)};

```

Comment:
Style:
Case inconsistency between `u` and `U` literals, or it doesn't matter?

Reply:
It seems that the standard wording consistently uses lowercase (`u`) in [rand], while MSVC STL consistently uses uppercase (`U`)...